<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAuditExplore</class>
 <widget class="QWidget" name="FormAuditExplore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Explorateur d'audit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/icons/icons/exploreAudit.png</normaloff>:/icons/icons/exploreAudit.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0" colspan="2">
         <widget class="QTreeView" name="treeView">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clic gauche pour sélectionner le noeud à étudier, double clic pour sélectionner le noeud de référence pour les comparaisons de certains tableaux et graphiques.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="comboListAudits"/>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButtonCollapseAll">
          <property name="text">
           <string>Replier tout</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonExpandAll">
          <property name="text">
           <string>Déplier tout</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="4">
         <widget class="QTextEdit" name="textEdit">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBoxDFRFDIRITable"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="comboBoxTable"/>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="saveTablesButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enregistrer les tableaux&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images.qrc">
            <normaloff>:/icons/icons/html.png</normaloff>:/icons/icons/html.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxDFRFDIRIOrYearsChart1"/>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBoxChart1"/>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="ZoomableQChartView" name="chartView1"/>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="saveChart1Button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enregistrer le graphique&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images.qrc">
            <normaloff>:/icons/icons/picture_save.png</normaloff>:/icons/icons/picture_save.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <widget class="QToolButton" name="saveChart2Button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enregistrer le graphique&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images.qrc">
            <normaloff>:/icons/icons/picture_save.png</normaloff>:/icons/icons/picture_save.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBoxChart2"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxDFRFDIRIOrYearsChart2"/>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="ZoomableQChartView" name="chartView2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableQChartView</class>
   <extends>QGraphicsView</extends>
   <header>zoomableqchartview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboListAudits</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>pushButtonExpandAll</tabstop>
  <tabstop>pushButtonCollapseAll</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
